### PR TITLE
Updating Documentation Test

### PR DESCRIPTION
**Pull request by Michael-Burke**
Updated Windows section of documentation to be more clear

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following information should get you up and running on either OS X, Windows
     - if make test fails, its most likely because pyyaml is not installed
 
 ### Windows 7
-(I don't use windows, but this should work)
+(NOT RELIABLE -- OLD INSTRUCTION -- DOESN'T WORK)
```
Comment (BadgerOps) [inline README.md]: Hey, thanks for this PR - would you mind either wiping out the windows section entirely or update it to work? Thanks!

**Review by BadgerOps — CHANGES_REQUESTED** [review]
Update or remove documentation lines, please!

File: README.md [hunk 2]
```diff
@@ -18,17 +20,16 @@ The following information should get you up and running on either OS X, Windows
       ```bash
       vagrant plugin install vagrant-hostmanager
       ```
-    c. OPTIONAL, for docker based testing: Install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
-
-    d. Ensure Git is installed [Download Git](http://git-scm.com/download/mac). If already installed via xcode tools, you should be ready to go.
+    C. Ensure Git is installed [Download Git](http://git-scm.com/download/mac). If already installed via xcode tools, you should be ready to go.
```
Comment (BadgerOps) [inline README.md]: can you lowercase the `C` and `D` to match the rest?

Comment (Michael-Burke) [reply README.md]: Done!

**Review by BadgerOps — APPROVED** [review]

**Comment by BadgerOps** [comment]
Thanks for this PR @Michael-Burke - looks good, I'll merge it now.

**Pull request merged**